packages/terraform/package.yaml

```
@@ -0,0 +1,53 @@
+---
+name: terraform
+description: |
+  Terraform enables you to safely and predictably create, change, and improve infrastructure. It is a source-available
+  tool that codifies APIs into declarative configuration files that can be shared amongst team members, treated as code,
+  edited, reviewed, and versioned.
+homepage: https://www.terraform.io/
+licenses:
+  - BUSL-1.1
+languages:
+  - Terraform
+categories:
+  - Formatter
```

```suggestion
  - Formatter
  - Runtime
```

I believe this is also the full terraform runtime